.github/workflows/ci.yml

```diff
@@ -57,6 +57,26 @@ jobs:
           - "php:codestyle"
         php-version:
           - 7.4
+  code-quality-frontend:
+    name: "Code quality frontend checks"
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: "Install modules"
+        working-directory: ./Resources/Private
+        run: yarn
+      -
+        name: "Run command"
+        working-directory: ./Resources/Private
+        run: "yarn lint:${{ matrix.command }}"
+    strategy:
```

Comment:
Please let's move this up so the reader already has read the matrix when the matrix variable is used.